x-pack/plugins/reporting/export_types/printable_pdf/server/lib/layouts/print.css

```diff
@@ -41,18 +41,21 @@ discover-app .discover-table-footer {
   display: none;
 }
 
+/**
+ * The global banner (e.g. "Help us improve Elastic...") should not print.
+ */
+
+#globalBannerList {
+  display: none;
+}
+
 /**
  *  Visualize Editor Tweaks
  */
+
 /* hide unusable controls */
-visualize-app
-  visualize
-  > :not(.vis-editor-content):not(.vis_editor)/* all non-content rows in interface */,
-visualize-app
-  visualize
-  .vis-editor-content
-  .default-vis-editor
-  > :not(.vis-editor-canvas):not(.vis_editor) {
+visualize-app visualize > :not(.vis-editor-content):not(.vis_editor), /* all non-content rows in interface */
```

Comment:
Since there is no `visualize` element anymore, that rule should not work. But to be honest, I had no idea what it tried to hide earlier (maybe TSVB editor?).